docs/ABOUT.md

```diff
@@ -1,14 +1,18 @@
 # About
 
-<!-- TODO: write document
+**_COBOL_** is a statically typed, imperative, and procedural programming language with support for object oriented programming. Despite its age, it's still used today in many banking and financial companies due to its default support for fixed-decimal arithmetic and file-processing ability, making it well suited for financial calculations and other business applications. 
```

Comment:
Hey @TriAttack238 I would recommend changing the "fixed-decimal arithmetic" to "decimal floating-point arithmetic"

This is a little bit confusing at first when looking for information on COBOL because most places will tell you that it uses fixed-point decimals, but both the 2014 and 2022 standards say that it must use IEEE 754 decimal floating-point arithmetic.

Reply:
Otherwise it looks good to me.

Reply:
It can do floating point, but a main reason it's still being used it because it's easy to do fixed point. It's a feature I want to point out because it points to the language's practicality and industry use.

Reply:
That sounds like a compatibility problem in the making, but I'll change it for now.